docs/support/troubleshooting.md

```diff
@@ -54,3 +54,14 @@ $ git branch --contains <tag name>
 $ git tag -d <tag name>
 $ git push origin :refs/tags/<tag name>
 ```
+
+## release not found in prereleases branch (e.g. `beta`) after rebase on `master`
```

Comment:
This is not specifically related to prereleases branches. This principle is true for any branch.

Reply:
semantic-release is using git notes for `master`, too?

Reply:
When a new tag is created yes. But if there is a tag without a note then we consider the version was published on the channel associated with `master`.
So if your version has been published on the channel associated with `master` (by default not specified so each plugin would use it's specific default such as `latest` for npm) and `next` then you will have a note with `{"channels:[null,"next"]"}`.